guide/additional-info/async-await.md

```diff
@@ -109,24 +109,18 @@ But since all of these react methods are started at the same time, it would just
 ```js
 client.on('message', message => {
 	if (message.content === `${prefix}react`) {
-		message.react('🇦').then(() => {
-			message.react('🇧').then(() => {
-				message.react('🇨').catch(error => {
-					// handle failure of the third react
-				});
+		message.react('🇦')            
+			.then(() => message.react('B'))
```

Comment:
These are the same smallcaps letters you use everywhere else.

Reply:
what do you mean by that? 🤔

Reply:
🇧 vs B